### PR TITLE
MWPW-158473: Top navigation wc

### PR DESCRIPTION
**Pull request by Axelcureno**
This PR adds the top navigation web component to merch at scale studio.
Screenshot 2024-10-04 at 2 24 56 PM

[MWPW-159622](https://jira.corp.adobe.com/browse/MWPW-159622)

Test URLs:
- Before: https://main--mas--adobecom.hlx.live/studio.html
- After: https://mwpw-158473--mas--adobecom.hlx.page/studio.html

### CODE REVIEW

File: studio/src/top-nav.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                        label="Help"
+                        placement="bottom-end"
+                </sp-top-nav-item>
+                </sp-top-nav-item>
+            </sp-top-nav>
+        `;
+    }
+}
+
+customElements.define('top-nav', TopNav);
```
Comment (yesil) [inline studio/src/top-nav.js:80]: eof

File: studio/src/top-nav.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        return html`
+            <sp-top-nav>
+                <sp-top-nav-item
+                    class="logo"
+                    size="l"
+                    href="#"
+                    label="Home"
+                    quiet
+                >
+                    <svg
```
Comment (yesil) [inline studio/src/top-nav.js:46]: can the SVG be an external file?

**Review by yesil — APPROVED** [review]
it would be nice to address this in its own ticket.
I would like us to discuss the requirements w.r.t. toolbars.
I'm ok to merge as it makes Studio visually more appealing but we will have to rework this.

I guess we can use https://www.adobe.com/federal/assets/svgs/adobe-logo.svg as icon.

Only ask is to externalize SVG files.

File: studio/src/top-nav.js (hunk 1 above)
Comment (Axelcureno) [reply studio/src/top-nav.js]: added

File: studio/src/top-nav.js (hunk 2 above)
Comment (Axelcureno) [reply studio/src/top-nav.js]: yes, done.

**Review by 3ch023 — APPROVED** [review]

**Pull request merged**